src/components/Annotation/Annotation.js

```diff
@@ -89,38 +69,17 @@ class Annotation extends React.PureComponent<Props> {
                     <div className='ba-annotation-content'>
                         <Avatar className='ba-annotation-avatar' {...createdBy} />
                         <div className='ba-annotation-text'>
-                            <div className='ba-annotation-headline'>
-                                <UserLink className='ba-annotation-user-name' {...annotator} />
-                                {createdAtTimestamp && (
-                                    <Tooltip
-                                        text={
-                                            <FormattedMessage
-                                                {...messages.annotationPostedFullDateTime}
-                                                values={{ time: createdAtTimestamp }}
-                                            />
-                                        }
-                                    >
-                                        <time className='ba-annotation-created-at'>
-                                            <ReadableTime
-                                                timestamp={createdAtTimestamp}
-                                                relativeThreshold={ONE_HOUR_MS}
-                                            />
-                                        </time>
-                                    </Tooltip>
-                                )}
-                                {hasDeletePermission ? (
-                                    <InlineDelete
-                                        id={id}
-                                        permissions={permissions}
-                                        message={<FormattedMessage {...messages.annotationDeletePrompt} />}
-                                        onDelete={onDelete}
-                                    />
-                                ) : null}
-                            </div>
+                            <AnnotationHeader
+                                id={id}
+                                permissions={permissions}
+                                onDelete={hasDeletePermission ? onDelete : noop}
```

Comment:
Minor: Can `hasDeletePermission` be evaluated within the header, since `permissions` are already passed in?